src/app/authorization/login/login.component.html

```diff
@@ -1,60 +1,92 @@
-<div class="card shadow" style="width: 25rem; margin-top: 80px;">
-    <div class="card-body">
-        <form [formGroup]="adminForm" (ngSubmit)="formSubmitted()" class="">
-            <div class="row g-0">
-                <div class="col text-center">
-                    <img src="assets/img/login.png" width="60px" height="60px">
-                    <br>
-                    <span class="fs-4">Login</span>
-                </div>
+<div class="container-fluid">
+  <div class="row align-items-center min-vh-100">
+    <div class="mx-auto col-10 col-sm-5 col-md-6 col-lg-5 col-xl-4 col-xxl-3">
+      <form
+        [formGroup]="adminForm"
+        (ngSubmit)="formSubmitted()"
+        class="rounded-4 mt-5 mb-5 bg-light"
+      >
+        <div class="col text-center p-3">
+          <img src="assets/img/login.png" width="60px" height="60px" />
```

Comment:
instad of giving width height here could you please change to external css

Reply:
yes , i changed to external css